### PR TITLE
School region stats

### PR DESCRIPTION
**Pull request by AnnaGoodman1**
updating region stats file - @Robinlovelace , please check What I have written is accurate as a description and correct if not. thanks!

### CODE REVIEW

**Review by Robinlovelace — APPROVED** [review]
Yes, that's correct. Many thanks for updating this important file.

File: school/lsoa/build_params_pct_region.csv [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-hereford-and-worcester,5,0.05,1,1,2725,77125,2019-01-06,0.06
-hertfordshire,5,0.05,1,1,6255,142315,2019-01-06,0.1
-humberside,5,0.05,1,1,3406,109557,2019-01-06,0.09
-isle-of-wight,5,0.05,1,1,578,14216,2019-01-06,0.01
-kent,5,0.05,1,1,9407,196336,2019-01-06,0.12
-lancashire,5,0.05,1,1,7877,172639,2019-01-06,0.28
-leicestershire,5,0.05,1,1,4330,121316,2019-01-06,0.06
-lincolnshire,5,0.05,1,1,2427,74651,2019-01-06,0.05
-liverpool-city-region,5,0.05,1,1,9485,185840,2019-01-06,0.13
-london,10,0.05,1,1,25202,695278,2019-01-06,0.34
```
Comment (Robinlovelace) [inline school/lsoa/build_params_pct_region.csv:27]: Yes, that's correct.

File: school/lsoa/region_stats.Rmd [hunk 2]
```diff
@@ -52,7 +52,7 @@ There were `r prettyNum(sum(z$all), big.mark = ",", scientific = F)` school chil
...
 
-Finally, the **Route Network (LSOA, clickable)** is built from all origin-destination fast routes within the region that contain at least `r region_build_param$minflow_rnet` children and have a fast route distance <5km (for primary school children)/<10km (for secondary school children). The pop-ups do not present exact numbers for segments containing 1 to 2 cyclists.
+Finally, the **Route Network (LSOA, clickable)** and the **Route Network (LSOA, image)** are  built from all origin-destination fast routes within the region. In the online version of the 'clickable' route network, we only present route segments that contain at least `r region_build_param$mingodutch_rnet` cyclists in the Go Dutch scenario.  All route segments are included in the versions that can be downloaded. The pop-ups and downloads do not present exact numbers for segments containing 1 to 2 cyclists.
```
Comment (Robinlovelace) [inline school/lsoa/region_stats.Rmd:55]: This is correct, except 

> All route segments are included in the versions that can be downloaded

Sould be replaced with

> All route segments for which the Go Dutch values are greater than 10 are included in the versions that can be downloaded

Comment (AnnaGoodman1) [reply school/lsoa/region_stats.Rmd:55]: ok, changing it

Comment (Robinlovelace) [reply school/lsoa/region_stats.Rmd:55]: Thanks.

**Pull request merged**